README.md

```diff
@@ -1,3 +1,31 @@
+* *Status:* Production
```

Comment:
I'll use the buttons, but I can't get a hold of Lisa. If she's doing any automated processing on READMEs, moving the lines she's asked for might break it.